lib/main.dart

```diff
@@ -6,184 +6,184 @@ import 'package:flutter/material.dart';
-          'mysterious pipe and wander into a magical new world. But when the '
-          'brothers are separated, Mario embarks on an epic quest to find '
-          'Luigi.',
-      voteAverage: 7.8,
-      genres: ['Animation', 'Family', 'Adventure', 'Fantasy', 'Comedy'],
-      countLikes: 0,
+    title: 'The Super Mario Bros. Movie',
+    originalTitle: 'The Super Mario Bros. Movie',
+    thumbnail: 'images/backdrop_mario_bros_movie.jpg',
+    poster: 'images/poster_mario_bros_movie.jpg',
+    overview: 'While working underground to fix a water main, Brooklyn '
+        'plumbers—and brothers—Mario and Luigi are transported down a '
+        'mysterious pipe and wander into a magical new world. But when the '
+        'brothers are separated, Mario embarks on an epic quest to find '
+        'Luigi.',
+    voteAverage: 7.8,
+    genres: ['Animation', 'Family', 'Adventure', 'Fantasy', 'Comedy'],
+    countLikes: 0,
   );
 
   const MyApp({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return  MaterialApp(
+    return MaterialApp(
         home: Scaffold(
-          appBar: AppBar(
-            backgroundColor: Colors.black87,
-            title: Text(marioBros.originalTitle),
-          ),
-          body: SafeArea(
-          top: true,
-          bottom: true,
-          left: true,
-          right: true,
-          child: Center(
-
-            child: ShowMovie(marioBros),
-          )
-          )
-        )
-    );
+            appBar: AppBar(
+              backgroundColor: Colors.black87,
+              title: Text(marioBros.originalTitle),
+            ),
+            body: SafeArea(
+                top: true,
+                bottom: true,
+                left: true,
+                right: true,
+                child: Center(
+                  child: ShowMovie(marioBros),
+                ))));
   }
+}
 
-
+class Movie {
+  final String title;
+  final String originalTitle;
+  final String thumbnail;
+  final String poster;
+  final String overview;
+  final double voteAverage;
+  final List<String> genres;
+  int countLikes;
+
+  Movie({
+    required this.title,
+    required this.originalTitle,
+    required this.thumbnail,
+    required this.poster,
+    required this.overview,
+    required this.voteAverage,
+    required this.genres,
+    required this.countLikes,
+  });
 }
 
-  class Movie {
-    final String title;
-    final String originalTitle;
-    final String thumbnail;
-    final String poster;
-    final String overview;
-    final double voteAverage;
-    final List<String> genres;
-    int countLikes;
-
-    Movie({
-      required this.title,
-      required this.originalTitle,
-      required this.thumbnail,
-      required this.poster,
-      required this.overview,
-      required this.voteAverage,
-      required this.genres,
-      required this.countLikes,
-    });
-  }
+class ShowMovie extends StatelessWidget {
```

Comment:
Lets find another name to this class , something that can represent an object , not a functionality

Reply:
done, i changed it to MovieDeck